### ISSUE TITLE
Arrow keys stop working apart from left

### ISSUE THREAD

**Issue by gcarreno** [cyberfilth/Axes-Armour-Ale#35]
Hey Chris (@cyberfilth),

Somehow 3 of the arrow keys are not working any more.
Only the left arrow key is working.

The `numpad` arrows still work as expected.

Cheers,
Gus

**Comment by cyberfilth** [comment]
Hi Gus,

Are you getting that issue when moving around the game map?
I'm testing on Ubuntu and don't seem to be getting the same problem.

regards,
Chris

**Comment by gcarreno** [comment]
Hey Chris (@cyberfilth),

Yeap, the issue is when I'm zipping around in the map.

Ok, now, that's rather odd! I'll have to see if it's an issue on my side.

I'll do some testing and report later :smile: 

Cheers,
Gus

**Comment by cyberfilth** [comment]
When you get a moment, let me know if you experience the same issue with this compiled version,

Regards,
Chris
[Axes.zip](https://github.com/cyberfilth/Axes-Armour-Ale/files/8168379/Axes.zip)

**Comment by gcarreno** [comment]
Hey Chris (@cyberfilth),

First of all, I'm really, really and really sorry for taking so long to answer this :frowning_face: 

The game was the first place I noticed that the arrows weren't working, but I then realized that the issue was showing itself on other applications.

I'm gonna close this issue in shame, for having bothered you with an issue that was going on on my computer and my computer alone.

I needed to reboot due to `libc` being updated and the issue just, poof, went away. Still don't have the foggiest clue of why it showed up in the first place.

Cheers,
Gus
